exotic.py

```diff
@@ -587,6 +593,38 @@ def plate_solution(fits_file, saveDirectory):
             return False
         time.sleep(5)
 
+#Checks if comparison star coordinates don't point to variable stars
+def variableStarCheck(rafile, decfile, compStarList):
+    validCoordList = []
+    errorMargin = 0.005556
+
+    #For every comparison star, convert into WCS coordinates and check if it's in WCS file before querying
+    #Also accounts for error margin of 20 arcseconds
+    for compStar in compStarList:
+        compStarCoords = compStar.all_pix2world()
+        raLowerBound = compStarCoords[0] - errorMargin
+        raUpperBound = compStarCoords[0] + errorMargin
+        decLowerBound = compStarCoords[1] - errorMargin
+        decLowerBound = compStarCoords[1] + errorMargin
+        if compStarCoords[0] in raList and compStarCoords[1] in decList:
+            validCoordList.append((compStarCoords[0], compStarCoords[1]))
+
+    #Use normalized Coordinates to search on SIMBAD to check if that star is variable
+    #Throws an error if invalid coordinates are inputted
+    for counter in range(0, len(validCoordList)):
+        try:
+            resultTable = Simbad.query_region(coord.SkyCoord(validCoordList[counter][0], validCoordList[counter][1], unit = (u.deg, u.deg)))
+        except TypeError:
+            print("Error: Invalid star coordinates. Checking next star... ")
+
+        #Check if star ID/name explicitly has V* in it
+        starName = resultTable['MAIN_ID'][0].decode("utf-8")
+        if "V*" in starName:
+            print("Comparison star " + starName + " is variable; selecting alternative star...")
+            compPixelCoords = validCoordList[counter].all_world2pix()
+            #validCoordList.pop(validCoordList[counter])
+            compStarList.remove(compPixelCoords)
+    return True
```

Comment:
the way your function runs, it will *always* return `True` - how does it behave differently if the star is variable or not? Don't you want to return instead the list of stars that are *not* variable?